ibis/backends/dask/client.py

```diff
@@ -0,0 +1,342 @@
+"""The dask client implementation."""
+
+from __future__ import absolute_import
+
+import re
+from functools import partial
+
+import dask.dataframe as dd
+import dateutil.parser
+import numpy as np
+import pandas as pd
+import toolz
+from multipledispatch import Dispatcher
+from pandas.api.types import DatetimeTZDtype
+from pkg_resources import parse_version
+
+import ibis.client as client
+import ibis.common.exceptions as com
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+import ibis.expr.schema as sch
+import ibis.expr.types as ir
+from ibis.backends.pandas.client import (  # noqa: F401
+    PANDAS_DATE_TYPES,
+    PANDAS_STRING_TYPES,
+    _ibis_dtypes,
+    _inferable_pandas_dtypes,
+    _numpy_dtypes,
+    convert_timezone,
+    from_numpy_dtype,
+    from_pandas_categorical,
+    from_pandas_tzdtype,
+    ibis_dtype_to_pandas,
+    ibis_schema_to_pandas,
+    infer_array,
+    infer_numpy_scalar,
+    infer_pandas_timestamp,
+)
+
+from .core import execute_and_reset
+
+infer_dask_dtype = pd.api.types.infer_dtype
```

Comment:
Does dask just pandas's dtypes for data types and schema?

Reply:
Yep

Reply:
kk, good to know. Thanks!